adapters/sharethrough/utils.go

```diff
@@ -208,6 +212,19 @@ func (u Util) gdprApplies(request *openrtb.BidRequest) bool {
 	return gdprApplies != 0
 }
 
+func (u Util) getUsPrivacySignal(request *openrtb.BidRequest) (ccpaSignal string) {
+	ccpaSignal = ""
+
+	if request.Regs != nil {
+		if jsonExtRegs, err := request.Regs.Ext.MarshalJSON(); err == nil {
```

Comment:
Consider using our built-in `ccpa.ReadPolicy(request)` helper in the `github.com/prebid/prebid-server/privacy/ccpa` package.